spark/scala-2.10/src/main/scala/org/apache/zeppelin/spark/SparkScala210Interpreter.scala

```diff
@@ -67,7 +67,7 @@ class SparkScala210Interpreter(override val conf: SparkConf,
     settings.usejavacp.value = true
     settings.classpath.value = getUserJars.mkString(File.pathSeparator)
     Console.setOut(interpreterOutput)
-    sparkILoop = new SparkILoop(null, new JPrintWriter(Console.out, true))
+    sparkILoop = new SparkILoop()
```

Comment:
is this the root of the build error?

Reply:
Yes

Reply:
Sorry, ignore. That was cherry-picked from https://github.com/apache/zeppelin/pull/2863 which has been merged. Let me rebase.

Reply:
Rebased. No longer here